lang/src/org/partiql/lang/syntax/PartiQLParser.kt

```diff
@@ -95,9 +95,12 @@ internal class PartiQLParser(
         val tokenIndexToParameterIndex = mutableMapOf<Int, Int>()
         var parametersFound = 0
         val tokens = CommonTokenStream(lexer)
```

Comment:
```suggestion
        val stream = CommonTokenStream(lexer).fill()
```

Reply:
`.fill()` is a void-returning function. But, agreed, it's worth changing to be more aesthetic.

Proposing the following instead:
```
val tokenStream = CommonTokenStream(lexer).also { it.fill() }
```

Committing in next revision.